nixos/tests/localsend.nix

```diff
@@ -0,0 +1,21 @@
+import ./make-test-python.nix (
+  { ... }:
+  {
+    name = "localsend";
+
+    nodes.machine =
+      { ... }:
+      {
+        imports = [ ./common/x11.nix ];
```

Comment:
Consider using Wayland instead?

Reply:
Does the Wayland module provide a graphical interface that we can wait for? The test relies on the command `machine.wait_for_window("LocalSend", 10)`. If the same functionality can be tested using Wayland, then I would also prefer to use Wayland instead of X11 for the test, but if I remember correctly, the Wayland module sets up a headless VM.

I do not have electicity at home at the moment so I will not be able to test using Wayland instead of X11 for at least a few days.

Reply:
> Does the Wayland module provide a graphical interface that we can wait for? The test relies on the command `machine.wait_for_window("LocalSend", 10)`. If the same functionality can be tested using Wayland, then I would also prefer to use Wayland instead of X11 for the test, but if I remember correctly, the Wayland module sets up a headless VM.

Exactly my point. I do not see it as a requirement to merge, but if @Pandapip1 would like to send a patch I am happy to commit it for you.

Reply:
Oh, I definitely don't see it as a requirement either. Certainly, it's not blocking.